app/views/devise/registrations/edit.html.haml

```diff
@@ -16,6 +16,8 @@
                     Currently waiting confirmation for: #{resource.unconfirmed_email}
                 = f.input :password, autocomplete: "off", hint: "leave it blank if you don't want to change it", placeholder: "Password", required: false, label: false
                 = f.input :password_confirmation, required: false, placeholder: "Password Confirmation", label: false
+                .form-group
+                  = f.input :country, priority: [ "GR", "HU", "RS" ], include_blank: "Select a country", label: false, required: true, class: 'form-control'
```

Comment:
It'll be good to add the list of prioritized countries to somewhere out of the view. There are several places we could put them but since they'll be used for both users and drawings we might as well put them somewhere global, so my suggestion would be in `config/initializers/constants.rb`:

```
TOP_COUNTRIES = %w(GR HU RS)
```